--- prev
+++ curr
@@ -116,13 +116,13 @@
 Machine-generated traffic (also called automated traffic) refers to the practice of sending automated queries to Google. This includes scraping results for rank-checking purposes or other types of automated access to Google Search conducted without express permission. Machine-generated traffic consumes resources and interferes with our ability to best serve users. Such activities violate our spam policies and the Google Terms of Service. 
 
-## Malware and malicious practices
-
-Google checks to see whether websites host malware or unwanted software that negatively affects the user experience. 
-
-Malware is any software or mobile application specifically designed to harm a computer, a mobile device, the software it's running, or its users. Malware exhibits malicious behavior that can include installing software without user consent and installing harmful software such as viruses. Site owners sometimes don't realize that their downloadable files are considered malware, so these binaries might be hosted inadvertently. 
-
-Unwanted software is an executable file or mobile application that engages in behavior that is deceptive, unexpected, or that negatively affects the user's browsing or computing experience. Examples include software that switches your home page or other browser settings to ones you don't want, or apps that leak private and personal information without proper disclosure. 
-
-Site owners should make sure they don't violate the Unwanted Software Policy and follow our guidelines. 
+## Malicious practices
+
+Malicious practices create a mismatch between user expectations and the actual outcome, leading to a negative and deceptive user experience, or compromised user security or privacy. 
+
+The following are common examples of malicious practices: 
+
+ * Malware is any software or mobile application specifically designed to harm a computer, a mobile device, the software it's running, or its users. Malware exhibits malicious behavior that can include installing software without user consent and installing harmful software such as viruses. Site owners sometimes don't realize that their downloadable files are considered malware, so these binaries might be hosted inadvertently. 
+ * Unwanted software is an executable file or mobile application that engages in behavior that is deceptive, unexpected, or that negatively affects the user's browsing or computing experience. Examples include software that switches your home page or other browser settings to ones you don't want, or apps that leak private and personal information without proper disclosure. Site owners should make sure they don't violate the Unwanted Software Policy and follow our guidelines. 
+ * Back button hijacking is when a site interferes with user browser navigation by manipulating the browser history or other functionalities, preventing them from using their back button to immediately get back to the page they came from. 
 
 ## Misleading functionality 