--- prev
+++ curr
@@ -10,4 +10,15 @@
 
 To get the latest Search Central documentation updates delivered to you, add the URL of this page to your feed reader, or add the feed URL directly: `https://developers.google.com/search/updates/search_docs_updates.rss`. 
+
+## September 2026
+
+September 8
+ 
+
+### Added documentation about regional differences in Search experience
+
+**What** : Added documentation about regional differences in Search experience, which includes information about search experiences available in certain countries, such as aggregator units, supplier units, and carousels. 
+
+**Why** : To help publishers, businesses, and aggregators learn about the different regional search features available and understand the eligibility criteria and how to participate. 
 
 ## August 2026