--- prev
+++ curr
@@ -10,4 +10,15 @@
 
 To get the latest Search Central documentation updates delivered to you, add the URL of this page to your feed reader, or add the feed URL directly: `https://developers.google.com/search/updates/search_docs_updates.rss`. 
+
+## October 2026
+
+October 1
+ 
+
+### Updated guidance on using generative AI content
+
+**What** : Updated the using generative AI content guide with information from the Search Quality Raters guidelines. 
+
+**Why** : To get our documentation in sync with our presentations we use at our developer events. 
 
 ## September 2026