--- prev
+++ curr
@@ -30,5 +30,5 @@
 ## Doorway abuse
 
-Doorway abuse is when sites or pages are created to rank for specific, similar search queries. They lead users to intermediate pages that are not as useful as the final destination. Examples of doorway abuse include: 
+Doorway abuse is when sites or pages are created to rank for specific, similar search queries. They lead users to intermediate pages that aren't as useful as the final destination. Examples of doorway abuse include: 
 
  * Having multiple websites with slight variations to the URL and home page to maximize their reach for any specific query
@@ -39,5 +39,5 @@
 ## Expired domain abuse
 
-Expired domain abuse is where an expired domain name is purchased and repurposed primarily to manipulate search rankings by hosting content that provides little to no value to users. Illustrative examples include, but are not limited to: 
+Expired domain abuse is where an expired domain name is purchased and repurposed primarily to manipulate search rankings by hosting content that provides little to no value to users. Illustrative examples include, but aren't limited to: 
 
  * Affiliate content on a site previously used by a government agency
@@ -137,5 +137,5 @@
 Scaled content abuse is when many pages are generated for the primary purpose of manipulating search rankings and not helping users. This abusive practice is typically focused on creating large amounts of unoriginal content that provides little to no value to users, no matter how it's created. 
 
-Examples of scaled content abuse include, but are not limited to: 
+Examples of scaled content abuse include, but aren't limited to: 
 
  * Using generative AI tools or other similar tools to generate many pages without adding value for users
@@ -156,21 +156,18 @@
  * Creating sites dedicated to embedding or compiling content, such as videos, images, or other media from other sites, without substantial added value to the user
 
-## Site reputation abuse
-
-Site reputation abuse is a tactic where third-party content is published on a host site mainly because of that host's already-established ranking signals, which it has earned primarily from its first-party content. The goal of this tactic is for the content to rank better than it could otherwise on its own. 
+## Site reputation policy
+
+The site reputation policy applies where third-party content is published on a host site mainly because of that host's already-established ranking signals, which it has earned primarily from its first-party content. The goal of this tactic is for the content to rank better than it could otherwise on its own. We have made changes applicable to the policy in the European Economic Area (EEA). 
 
 _Third-party content_ is content that's created by an entity that's separate from the established host site. Examples of separate entities include users of that site, freelancers, white-label services, and content created by people not employed directly by the host site.
 
-Having third-party content alone isn't a violation of the site reputation abuse policy; it's only a violation if the third-party content is published on a host site mainly because of that host site's already-established ranking signals. Examples of site reputation abuse include, but are not limited to: 
+Having third-party content alone isn't inconsistent with the site reputation policy; it's only inconsistent if the third-party content is published on a host site mainly because of that host site's already-established ranking signals. 
+
+Examples inconsistent with the site reputation policy include, but aren't limited to: 
 
  * An educational site hosting a page about sponsored reviews of payday loans written by a third-party that distributes the same page to other sites across the web
- * A medical site hosting a third-party advertising page about "best casinos" that readers wouldn't expect and that's being placed on the site to rank better due to the established site's ranking signals
- * A movie review site hosting third-party pages about topics that would be confusing to users to find on a movie review site (such as "ways to buy followers on social media sites", the "best fortune teller sites", and the "best essay writing services")
- * A news site hosting coupons provided by a third-party white-label service where the main reason for publishing the coupons on the news site is to capitalize on the news site's reputation
- * An established first party site branches out into a new area primarily using freelance content because this content will rank better on the first-party site than it would have otherwise
-
-If you're hosting pages that violate this policy, learn how to correct this issue. 
-
-Examples that are **NOT** considered site reputation abuse include: 
+ * A medical site hosting a low-quality, third-party advertising page about "best casinos" that isn't integrated with the site, and that's being placed on the site to rank better due to the established site's ranking signals
+
+Examples that are **NOT** considered inconsistent with the site reputation policy include: 
 
  * Wire service or press release service sites
@@ -180,9 +177,82 @@
  * Third-party content (for example, "advertorial" or "native advertising" type pages) where the purpose is to share content directly to readers (such as through promotion within the publication itself), rather than hosting the content to manipulate search rankings
  * Using affiliate links throughout a page, with links treated appropriately, or embedding third-party ad units throughout a page
- * Coupons that are sourced directly from merchants and other businesses that serve consumers
+
+These examples are non-exhaustive. There will be other instances where we don't take action. 
+
+Google generally applies a presumption that individual pages (including new pages) match the overall quality of other pages on the domain. If we detect that a portion of your site may be out of line with this policy, a site will be subject to human review. As part of this review, if the site is found to be inconsistent with the policy, the consequences for the way the site's pages appear in search results will vary depending on the location of users. 
+
+ * **Outside theEEA**: If a site is found to be out of line with this policy, the relevant pages may be subject to a manual action when they appear in Search results shown to users outside the EEA.
+ * **Within the EEA** : When pages appear in Search results shown to users within the EEA, the relevant pages may be categorized as separate from the main domain but won't be subject to the impact of manual action. This will allow the different parts of the site to rank independently of each other, on their own merits. This ensures that content is ranked in a consistent way against other content of the same nature (for example, casino content ranks against other casino content), so that users get the best possible results for their query.
+
+If your site is affected in this way, we will notify you in the Manual actions report and in the Search Console message center. All sites will have the opportunity to address this issue or appeal via a reconsideration request. Eligible sites will, following the reconsideration request, also have the opportunity to bring disputes to mediation. 
+
+### More detailed guidance
+
+On the rare occasions where we perform a human review, our overarching goal is to determine whether content on the relevant portion of the site is created with sufficient input, editorial oversight, or contribution from the host site to be considered fully integrated with the main site. This ensures that a page's rank reflects the way it is presented to and understood by users. 
+
+This review applies globally and takes into account a number of objective factors, aimed at understanding the degree to which the host domain exercises control over the page content. These may include: 
+
+ * **How the content is presented** , meaning are the graphic design, formatting, typography and UX features of the content consistent with the host domain?
+ * **The quality of the content** , meaning, are there quality issues present on the page that don't appear on the main domain and otherwise suggest a departure from the standard of quality of the main domain?
+ * **Its stated or implied authorship** , meaning, is there an explicit acknowledgement of ownership or responsibility for the content? Are there indications that contest the stated authorship of the content?
+ * **Does the content appear on multiple other sites in identical or near-identical form** , meaning, the same identical or near-identical content appears on multiple other sites?
+
+Importantly, not one of these factors is either necessary or sufficient on its own to determine whether a given portion of a site is in line with this policy. Based on the particular situation, some factors may be more relevant than others, and other evidence may need to be taken into account (for example, if certain pages on a site are part of third-party marketing material for manipulating Search ranking). Here are some examples of how we might approach this: 
+
+ 1. **Not likely to take action: integrated coupons' deals section**
+
+A publisher hosts a coupons and deals section in partnership with a specialist provider, on a different CMS than the main site, under a sub-folder fully integrated in the homepage and in the snippets available in the publisher's articles. The coupons are organized under different categories specific for that publisher (for example, special deals, beauty, home appliances, clothes, local shops), even if some of the codes may be available elsewhere.
+
+The commercial communications character is clearly disclosed and the section provides disclaimers identifying the publisher as editorially responsible in collaboration with a partner (for example, provided by the publisher or dedicated editing team of the publisher in cooperation with the partner). Some codes in the section are cross-referred in other editorial content of the publisher or newsletters edited by the publisher, and there has been sufficient curation of the coupons and deals showing that content represents a differentiated, joint editorial effort between the publisher and the commercial partner. The section is easily navigable from the main page or from main sections of the main page (for example, the "Trends" section). Problems with the codes or offers provided can be reported to the publisher through an easily accessible link on the page and are treated in accordance with general publisher's contact standards.
+
+Unlikely to take action: even though the section is created in collaboration with a third party, the content — an organized repository of coupons and deals generally available on the market — complements and is integrated in the publication. Users are made aware of the responsible persons for it and a contact point in case of problems, in accordance with the publisher's general editorial standards.
+
+ 2. **Likely to take action: unauthored affiliate article not integrated in publication**
+
+A globally recognized business publication hosts an article including links to a marketplace selling CBD oils. Neither the author of the article nor the responsible editor is identified, nor are there disclaimers about the commercial character of the content. The article isn't part of any of the thematic sections of the publication, and no links or menu are available to reach the article from the main page or thematic sections of the publication. While the page is made available on a portion of the publisher's main site, there is evidence that the specific article is purely duplicating the same content provided by a third-party digital marketplace illustrating such a product.
+
+In this case, and without prejudice to any other policy that may be applicable, because the content is unrelated to the editorial sections of the publisher, and because the presentation, UX features, the quality don't match the business publication, including appropriate disclaimer or marks of commercial character of the content, and authorship and the responsible editor are unclear, but rather content is copied from another site, we would likely take action for search results that appear outside the EEA.
+
+ 3. **Unlikely to take action: authored, original, affiliate article from freelancer**
+
+A news site branches out to develop a new cooking section, including affiliate links to grocery stores and cookware. The content is produced by a freelancer interviewing international guest celebrity chefs, and there is evidence of editorial oversight by the host publication: for example, the site is branded consistently, contains clear statements of editorial responsibility of the news site, and the freelancer is identified as author.
+
+The same freelancer also provides similar content to other publications and publishers, but the selection of interviews or recipes provided is specific to the news site at stake, even if the affiliate links are similar or the same as those appearing in other publications.
+
+In this case, even though the content is authored by a third party also working for other websites, it's primarily unique content generated for the host site and its end users. Authorship, editorial responsibility, and affiliate links are clearly marked, and the formatting and presentation of the content are consistent with the rest of the publisher's content. It's unlikely that we'd take action in this case.
+
+### FAQ
+
+#### Will manual action taken outside the EEA affect the ranking of my site within the EEA?
+
+No. Manual actions involving the site reputation policy outside the EEA only affect results shown to users outside the EEA, and not those shown to users within the EEA. The fact that part of a site is subject to a manual action under this policy outside the EEA isn't used as a signal for ranking that content within the EEA.
+
+There's no obligation to apply a `noindex` tag to content that is subject to a manual action outside the EEA and failure to do so isn't taken into account in ranking that content within the EEA. Nor is such failure considered an attempt to evade or circumvent this policy or a repeated violation.
+
+#### A portion of my site was previously subject to manual action under this policy in the EEA. What happens now?
+
+Google will lift all previous manual actions taken under this policy for pages appearing in search results for users in the EEA. This means that those pages will no longer be demoted or subject to any kind of manual penalty in search results shown to users in the EEA.
+
+Going forward, those pages may be categorized as separate from the main domain and ranked on their own merits, but this isn't automatic.
+
+Our systems ensure that pages that were previously subject to a manual action under this policy rank fairly and aren't disadvantaged. The fact that a page was previously subject to a manual action under this policy isn't used as a ranking signal when ranking that page.
+
+#### What happens when a part of my site is categorized as separate from the main domain for being out of line with this policy?
+
+This categorization tells our systems that the presumption that we generally apply globally — which is that individual pages (including new pages) match the overall quality of other pages on the domain — no longer applies.
+
+This doesn't mean that the separate portion of the site immediately loses the ranking signals of the main site. Moreover, over time, our ranking systems learn to rank these parts of a site independently. This may lead to changes in the way that each part of the site ranks, including as a result of improvements in site-wide signals for one part of the site.
+
+The fact that this categorization has been applied isn't used as a ranking signal.
+
+#### What can I do if I disagree with the action taken on the domain?
+
+For websites in the EEA, we have implemented a new reconsideration request process through which we commit to answer requests within a short timeframe and provide more details on the reasoning.
+
+In the EEA, you can also make use of alternative dispute resolution.
 
 ## Sneaky redirects
 
-Redirecting is the act of sending a visitor to a different URL than the one they initially requested. Sneaky redirecting is the practice of doing this maliciously in order to either show users and search engines different content or show users unexpected content that does not fulfill their original needs. Examples of sneaky redirects include: 
+Redirecting is the act of sending a visitor to a different URL than the one they initially requested. Sneaky redirecting is the practice of doing this maliciously in order to either show users and search engines different content or show users unexpected content that doesn't fulfill their original needs. Examples of sneaky redirects include: 
 
  * Showing search engines one type of content while redirecting users to something significantly different
@@ -228,5 +298,5 @@
 ### Policy circumvention
 
-If a site continues to engage in actions intended to bypass our spam policies or content policies for Google Search, we may take appropriate action which may include restricting or removing eligibility for some of our search features (for example, Top Stories, Discover) and taking broader action in Google Search (for example, removing more sections of a site from Search results). Circumvention includes but is not limited to: 
+If a site continues to engage in actions intended to bypass our spam policies or content policies for Google Search, we may take appropriate action which may include restricting or removing eligibility for some of our search features (for example, Top Stories, Discover) and taking broader action in Google Search (for example, removing more sections of a site from Search results). Circumvention includes but isn't limited to: 
 
  * Using existing or creating new subdomains, subdirectories, or sites with the intention of continuing to violate our policies